pages/faq.tsx

```diff
@@ -29,132 +32,212 @@ export default function FAQ() {
+                  </li>
+                  <li>
+                    Stories should not have personal identifying information about other people
+                    impacted by COVID-19 without their consent
+                  </li>
                 </ol>
-                
-                <p>A story that does not meet these guidelines will not be shared publicly. If you have any questions or concerns, reach out to <a href="mailto:info@mycovidstory.ca">info@mycovidstory.ca</a>.</p>
+
+                <p>
+                  A story that does not meet these guidelines will not be shared publicly. If you
+                  have any questions or concerns, reach out to{' '}
+                  <a href="mailto:info@mycovidstory.ca">info@mycovidstory.ca</a>.
+                </p>
               </AccordionPanel>
             </AccordionItem>
 
             <AccordionItem>
               <h2>
                 <AccordionButton>
                   <Box flex="1" textAlign="left">
-                    <Heading as="h2" size="sm">Who can access private data?</Heading>
+                    <Heading as="h2" size="sm">
+                      Who can access private data?
+                    </Heading>
                   </Box>
                   <AccordionIcon />
                 </AccordionButton>
               </h2>
               <AccordionPanel pb={4}>
-                You have full control over who your information will be shared with — nobody else will have access to any personally identifiable information. We take your privacy seriously and have taken this into consideration at every step. The site does not use any third-party tracking or analytics tools.
+                You have full control over who your information will be shared with — nobody else
+                will have access to any personally identifiable information. We take your privacy
+                seriously and have taken this into consideration at every step. This site does not
+                use cookies or any third-party tracking tools. We use{' '}
+                <a
+                  href="https://app.usefathom.com/share/xnknpyhv/mycovidstory.ca#/"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  Fathom
+                </a>{' '}
+                to generate simple traffic metrics. Fathom does not track any personally
+                identifiable information.
               </AccordionPanel>
             </AccordionItem>
 
             <AccordionItem>
               <h2>
                 <AccordionButton>
                   <Box flex="1" textAlign="left">
-                    <Heading as="h2" size="sm">Why might the media want to follow-up with storytellers? Is MyCovidStory.ca a direct line to media coverage?</Heading>
+                    <Heading as="h2" size="sm">
+                      Why might the media want to follow-up with storytellers? Is MyCovidStory.ca a
```

Comment:
I think we have most of the site normalized to have COVID in all caps wherever it appears (except for like a text representation of the URL)

Reply:
@JamiesonRoberts based off other places this is standardized? I think it's ok because it's our URL but otherwise agree.

Reply:
Yeah thats fair!